### PR TITLE
(🎁) run the ci on pull requests

### PR DESCRIPTION
**Pull request by KotlinIsland**
### Description
Run ci on pull requests

### Expected Behavior
Formatting / lint / tests will make the PR fail

- extracted from #469

### CODE REVIEW

**Comment by ymyzk** [comment]
@KotlinIsland Could you use the new PR template so that people can understand the necessary summary, motivation, context, and expected bahavior of the change?

**Comment by KotlinIsland** [comment]
> @KotlinIsland Could you use the new PR template so that people can understand the necessary summary, motivation, context, and expected behavior of the change?

Okay, I've updated the description as specified.

**Comment by ymyzk** [comment]
Thanks for updating the ticket description.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -53,6 +53,7 @@ jobs:
       - test_app
       - test_frontend
     runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
```
Comment (ymyzk) [inline .github/workflows/ci.yml:56]: As this project still uses `master` as the default branch name, this `if` doesn't have any effects. I'll fix this by myself.

**Pull request merged**